### PR TITLE
Fix vet warning

### LINKED ISSUE #11768: Fix vet warnings in integration-cli
You can find them with go vet tool:
```
go vet integration-cli/*
```

### PR DESCRIPTION
**Pull request by moomou**
Fixes https://github.com/docker/docker/issues/11768

Signed-off-by: Paul ppymou@gmail.com

### CODE REVIEW

File: integration-cli/docker_cli_run_test.go [hunk 1]
```diff
@@ -2857,14 +2857,16 @@ func TestRunAllowPortRangeThroughExpose(t *testing.T) {
 		t.Fatal(err)
 	}
 	var ports nat.PortMap
-	err = unmarshalJSON([]byte(portstr), &ports)
+	if err = unmarshalJSON([]byte(portstr), &ports); err != nil {
```
Comment (LK4D4) [inline integration-cli/docker_cli_run_test.go]: Not sure what this mean. Should be:
```
if err := unmarshalJSON([]byte(portstr), &ports); err != nil {
```

**Comment by LK4D4** [comment]
LGTM

File: integration-cli/docker_cli_run_test.go (hunk 1 above)
Comment (tiborvass) [inline integration-cli/docker_cli_run_test.go]: `err :=`

**Comment by crosbymichael** [comment]
LGTM

@moo-mou thanks!

**Pull request merged**